index.html

```diff
@@ -4,181 +4,184 @@
+                  <h2>Kaja</h2>
+                  <p class="title">Design-team Leader</p>
+                  <p>lorem ipsum ipsum lorem.</p>
+                  <p>designers@theothers.com</p>
+              </div>
+          </div>
+      </div>
+
+      <div class="column">
+          <div class="card">
+              <img src="https://cdn.shopify.com/s/files/1/1061/1924/products/Monkey_Face_Emoji_large.png?v=1480481036" alt="Marcin" style="width:100%">
+              <div class="container">
+                  <h2>Marcin</h2>
+                  <p class="title">Tester-team Leader</p>
+                  <p>lorem ipsum ipsum lorem.</p>
+                  <p>testers@theothers.com</p>
+              </div>
+          </div>
+      </div>
+
+      <div class="column">
+          <div class="card">
+              <img src="https://cdn.shopify.com/s/files/1/1061/1924/files/Hear_No_Evil_Monkey_Emoji.png?9898922749706957214" alt="Norbert" style="width:100%">
+              <div class="container">
+                  <h2>Norbert</h2>
+                  <p class="title">Developers-team Leader</p>
+                  <p>lorem ipsum ipsum lorem.</p>
+                  <p>developers@theothers.com</p>
+              </div>
+          </div>
+      </div>
+
+      <div class="column">
+          <div class="card">
+              <img src="https://cdn.shopify.com/s/files/1/1061/1924/files/Speak_No_Evil_Monkey_Emoji.png?9898922749706957214" alt="Piotr" style="width:100%">
+              <div class="container">
+                  <h2>Piotr</h2>
+                  <p class="title">Delivery Manager</p>
+                  <p>lorem ipsum ipsum lorem.</p>
+                  <p>delivery@theothers.com</p>
+              </div>
+          </div>
+      </div>
+  </div>
             </div>
         </div>
     </div>
 
-    <div class="column">
-        <div class="card">
-            <img src="https://cdn.shopify.com/s/files/1/1061/1924/files/Hear_No_Evil_Monkey_Emoji.png?9898922749706957214" alt="Norbert" style="width:100%">
-            <div class="container">
-                <h2>Norbert</h2>
-                <p class="title">Developers-team Leader</p>
-                <p>lorem ipsum ipsum lorem.</p>
-                <p>developers@theothers.com</p>
-            </div>
-        </div>
-    </div>
-    <div class="column">
-        <div class="card">
-            <img src="https://cdn.shopify.com/s/files/1/1061/1924/files/Speak_No_Evil_Monkey_Emoji.png?9898922749706957214" alt="Piotr" style="width:100%">
-            <div class="container">
-                <h2>Piotr</h2>
-                <p class="title">Delivery Manager</p>
-                <p>lorem ipsum ipsum lorem.</p>
-                <p>delivery@theothers.com</p>
-            </div>
-        </div>
-    </div>
-</div>
+        <!-- obrazek dotyczy tla -->
+        <img class="background-image" src="https://upload.wikimedia.org/wikipedia/commons/8/8d/Danzig_Langgasse.jpg" alt="Gdańsk tramwaj 1906 rok">
```

Comment:
Yo :D generalnei do każdego obrazka mamy class: background image trzeba bedzie to pozmieniac w niedziele :) no i ten obrazek nie ma nic w css przez co jest duży- ja bym zmienila szerokosc przynajmniej, zeby w bok nie trzeba bylo przewijac